docs/source/conf.py

```diff
@@ -181,6 +181,11 @@
 # (source start file, name, description, authors, manual section).
 man_pages = [(master_doc, project, project + " Documentation", [author], 1)]
 
+# -- Options for linkcheck builder ----------------------------------------------
+# regex pattern 0: allow linking to a specific selection state in
+#  tensorboard.dev links while continuing to validate the base experiment link
+linkcheck_anchors_ignore = ["scalars.*&runSelectionState.*"]
+
```

Comment:
@Borda To fix the linkcheck error in the build-docs test (I'm pretty sure this is the failing check you were alluding to) I've enhanced the linkcheck builder config in conf.py to allow linking to a specific selection state in tensorboard.dev experiments while continuing to validate the base experiment link. I think this will be useful for other tutorials in the future.